libraries/apollo-runtime-java/api/apollo-runtime-java.api

```diff
@@ -60,3 +69,68 @@ public abstract interface class com/apollographql/apollo3/runtime/java/intercept
 	public abstract fun proceed (Lcom/apollographql/apollo3/api/ApolloRequest;Lcom/apollographql/apollo3/runtime/java/ApolloCallback;)V
 }
 
+public class com/apollographql/apollo3/runtime/java/internal/MapUtils {
+	public fun <init> ()V
+	public static fun entry (Ljava/lang/Object;Ljava/lang/Object;)Ljava/util/Map$Entry;
+	public static fun mapOf ([Ljava/util/Map$Entry;)Ljava/util/Map;
```

Comment:
Nitpicking: I have the feeling that `ImmutableMapBuilder` would be somewhat more idiomatic (but not sure). In all cases, can you add `com/apollographql/apollo3/runtime/java/internal/` to the ignored packages of apiDump so that it doesn't appear here?

Reply:
Good call!

Reply:
Turns out we have one already: https://github.com/apollographql/apollo-kotlin/blob/05f2a3295517fb7a64c2259c38b453a37f3c4d19/libraries/apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/ImmutableMapBuilder.kt#L6. (I thought it was a JDK thing but it's not, it's coming [from guava](https://guava.dev/releases/19.0/api/docs/com/google/common/collect/ImmutableMap.Builder.html) actually). Maybe reuse that? Or is it too verbose?

Reply:
Let's do this for v5 😂